### ISSUE TITLE
Sentry generating a lot of "not implemented" errors

### ISSUE THREAD

**Issue by Greewi** [getsentry/sentry-cordova#146]
Hello,

We juste integrate sentry in one of our cordova projects and we get a lot of errors generated by the Sentry's code in the javascript side of our application. The error message is "not implemented" (litteraly that string).

We use the latest version (0.16.0).

Here is a the callstack of the first occurence of the errors :
callbackFromNative (cordova.js:289)
processMessage (cordova.js:1129)
processMessages (cordova.js:1152)
Promise.then (async)
nextTick (cordova.js:956)
androidExec (cordova.js:998)
(anonymous) (sentry-cordova.bundle.js:7065)
(anonymous) (sentry-cordova.bundle.js:7051)
step (sentry-cordova.bundle.js:214)
(anonymous) (sentry-cordova.bundle.js:195)
(anonymous) (sentry-cordova.bundle.js:188)
__awaiter (sentry-cordova.bundle.js:184)
CordovaBackend._nativeCall (sentry-cordova.bundle.js:7048)
(anonymous) (sentry-cordova.bundle.js:7089)
(anonymous) (sentry-cordova.bundle.js:1821)
(anonymous) (sentry-cordova.bundle.js:1820)
sentryWrapped (sentry-cordova.bundle.js:4743)
setTimeout (async)
(anonymous) (sentry-cordova.bundle.js:5117)
Scope._notifyScopeListeners (sentry-cordova.bundle.js:1819)
Scope.addBreadcrumb (sentry-cordova.bundle.js:2001)
Hub.addBreadcrumb (sentry-cordova.bundle.js:2292)
Breadcrumbs.addBreadcrumb (sentry-cordova.bundle.js:5636)
(anonymous) (sentry-cordova.bundle.js:5297)
<Our function that makes a console.log('Ok')>

**Comment by Greewi** [comment]
Note : we use sentry onpremise.

**Comment by faugusztin** [comment]
When i set a breakpoint for cordova.exec inside the plugin's JS, i seen call to setExtraContext method of Sentry plugin.
When i look at iOS version of plugin, i see such methods:
```- (void)sendEvent:(CDVInvokedUrlCommand *)command;
- (void)addBreadcrumb:(CDVInvokedUrlCommand *)command;
- (void)setUserContext:(CDVInvokedUrlCommand *)command;
- (void)setTagsContext:(CDVInvokedUrlCommand *)command;
- (void)setExtraContext:(CDVInvokedUrlCommand *)command;
- (void)clearContext:(CDVInvokedUrlCommand *)command;
```

Android version of plugin seems to have only the install method, for everything else it returns "not implemented".

We are either both missing something in installation/configuration steps or nobody else was bothered by those messages on Android version of plugin.

**Comment by faugusztin** [comment]
Upon closer inspection v0.14 had empty catch after calling these methods, for example https://github.com/getsentry/sentry-cordova/blob/v0.14.0/src/js/backend.ts#L112 :
```
this.nativeCall('setExtraContext', (scope as any).extra).catch(() => {
      // We do nothing since scope is handled and attached to the event.
      // This only applies to android.
    });
```

This got replaced by a forget method call from @sentry/utils in v0.15 https://github.com/getsentry/sentry-cordova/blob/v0.15.0/src/js/backend.ts#L125 :
```
forget(this._nativeCall('setExtraContext', (internalScope as any)._extra));
```
Problem is that forget function in @sentry/utils is defined with console.log, unlike the original code - https://github.com/getsentry/sentry-javascript/blob/master/packages/utils/src/async.ts:
```
/**
 * Consumes the promise and logs the error when it rejects.
 * @param promise A promise to forget.
 */
export function forget(promise: Promise<any>): void {
  promise.catch(e => {
    // TODO: Use a better logging mechanism
    console.error(e);
  });
}
```

The workaround is to use sentry-cordova version 0.14.

Solution is one of the following:

- implementation of backend.ts to revert to original implementation of the catch method
- providing an no-op implementation in SentryCordova.java of the methods available in iOS version of plugin (addBreadcrumb, setUserContext, setTagsContext, setExtraContext, clearContext) instead of returning an 'not implemented' error.

**Comment by HazAT** [comment]
This should be fixed in `0.16.2`, thanks for reporting
